### ISSUE TITLE
Périmètre d'une procédure secondaire : confusions avec le CTA "communes concernées" : Saulxerotte (54)

### ISSUE THREAD

**Issue by CeliaVermicelli** [MTES-MCT/Docurba#504]
FYI @Octaviedebs 

Cette [FRP](https://docurba.beta.gouv.fr/frise/d03520c4-4d9b-4b64-bfc9-2afd32b61e57)  devrait avec les 38 communes de l'EPCI dans la liste des communes concernées. 

Cf [feedback utilisateur](https://www.notion.so/docurba/Choix-du-p-rim-tre-d-une-proc-dure-91bd8ba734a249e7ad861308e005cc85?pvs=4) sur une autre EPCI qui avait le même cas, [conversation avec hermance et françois](https://docurba.slack.com/archives/C05RDPJPP3L/p1711473938320499) et la possibilité de changer "communes concernées" par "périmètre du document d'urbanisme". 

-> est-ce que c'est possible de modifier le périmètre de [cette FRP](https://docurba.beta.gouv.fr/frise/d03520c4-4d9b-4b64-bfc9-2afd32b61e57) en sélectionnant l'ensemble des communes de l'EPCI svp ?

**Comment by HermanceGauth** [comment]
@CeliaVermicelli est-ce que cette issue est sur le changement de wording normalement pris en compte dans ces maquettes (https://www.figma.com/design/4W463mFBKoDzf4mgJAhW9y/Docurba-(V2)?node-id=3605-16156&t=89GjWqtnMdiiUimo-0) ou y a-t-il une autre action à faire? une modification manuelle du périmètre d'une procédure créée? un cas d'usage de ce qu'on a vu hier avec @julienzdocurba ?

**Comment by CeliaVermicelli** [comment]
### Sur le changement de wording : 

- je ne suis pas sûre d'avoir retrouvé le changement sur les maquettes ; via le lien partagé je retrouve "communes concernées". 
- la proposition retenue pour ce changement de wording était bien : _Périmètre du document d'urbanisme_ FYI @julienzdocurba 

<img width="976" alt="Capture d’écran 2024-05-30 à 12 07 28 PM" src="https://github.com/MTES-MCT/Docurba/assets/111045572/3dd5c2b8-35a9-4383-9f79-f1aee999e0fa">



### Sur le changement de périmètre demandé : 

* Effectivement, on aurait voulu idéalement ne pas avoir à demander à la collectivité/DDT de re-créer la proc. et tout re-renseigner. mais si besoin on peut s'en charger quand même. 
* **Cas d'usage mentionné hier** : erreur de saisie du périmètre au moment de la création de la procédure car n'avait pas compris ce qui était demandé FYI @julienzdocurba 

Voici la proc : 
<img width="1007" alt="Capture d’écran 2024-05-30 à 12 13 51 PM" src="https://github.com/MTES-MCT/Docurba/assets/111045572/b9c27086-33d0-4080-9eaf-4be02cebf7b3">


On a un PLUI opposable - 38 communes 
ses proc. secondaires : 
* MS nº4 - 1 communes seulement 
* MS nº3 - 3 communes seulement

-> **il faudrait changer le périmètre de ces deux procédures sondaires et sélectionner l'ensemble des communes du périmètre de cet EPCI.**

**Comment by julienzdocurba** [comment]
> je ne suis pas sûre d'avoir retrouvé le changement sur les maquettes ; via le lien partagé je retrouve "communes concernées".
> la proposition retenue pour ce changement de wording était bien : Périmètre du document d'urbanisme

Oui, c'est ici dans la proposition sur Figma: https://www.figma.com/design/4W463mFBKoDzf4mgJAhW9y/Docurba-(V2)?node-id=5545-82612&t=BTKtYS3KOzpsH1fK-0

>erreur de saisie du périmètre au moment de la création de la procédure car n'avait pas compris ce qui était demandé

Super merci!

**Comment by Octaviedebs** [comment]
Donc **action dev :** =

On a un PLUI opposable - 38 communes
ses proc. secondaires :
MS nº4 - 1 communes seulement
MS nº3 - 3 communes seulement
**-> il faudrait changer le périmètre de ces deux procédures sondaires et sélectionner l'ensemble des communes du périmètre de cet EPCI.**
